### ISSUE TITLE
packit-stg @ MP+ follow-up

### ISSUE THREAD

**Issue by jpopelka** [packit/deployment#509]
Nice to have once we move packit(-stg) to MP+ (#470):
- [ ] Use [internal Splunk image](https://source.redhat.com/departments/it/devit/it-infrastructure/itcloudservices/itocp/it_paas_kb/logging_to_splunk_on_managed_platform) because [our image](https://github.com/packit/fluent-plugin-splunk-hec)
  - [will reach End of Support on January 1, 2024](https://github.com/packit/fluent-plugin-splunk-hec#end-of-support)
  - tries to send the logs to [http-inputs-rhcorporate.splunkcloud.com](https://source.redhat.com/departments/it/splunk/splunk_wiki/splunk_fy20_architecture_blueprint#splunk-http-event-collectors-hec-) which is blocked by the firewall. The firewall allows `http-inputs-osdsecuritylogs.splunkcloud.com` which I tried, but our token doesn't work with it.
- [ ] Create a [Flower](https://github.com/mher/flower/blob/master/.github/workflows/docker.yml#L33) PR to [push the image also to Github Packages/Registry](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-docker-hub-and-github-packages)

**Comment by jpopelka** [comment]
Moved into the main MP+ ticket packit/private#13
